### PR TITLE
[Documentation][Bundles] SyliusGridBundle docs update

### PR DESCRIPTION
**Pull request by TheMadeleine**
| Q               | A
| --------------- | ---
| Doc fix?        | yes
| New docs?    | no
| BC breaks?      | no
| Related tickets | partially #5275 
| License         | MIT

### CODE REVIEW

**Comment by ylastapis** [comment]
@TheMadeleine What about custom actions on the grid ?
Is this even possible ?

You should have a little sentence for actions in "main" section and the ones in "item" section :)

Great thanks for your job.

File: docs/bundles/SyliusGridBundle/filters.rst [hunk 1]
```diff
@@ -1,144 +1,41 @@
...
                         type: string
-                        options:
-                            fields: [username, e-mail, firstName, lastName]
 
 The filter allows the user to select following search options:
 
 * contains
 * not contains
+* equal
 * starts with
```
Comment (TheMadeleine) [inline docs/bundles/SyliusGridBundle/filters.rst]: There is no ``not equal`` :)

**Comment by TheMadeleine** [comment]
@ylastapis to be honest I'm not sure about the custom actions on grids. Let me call @pjedrzejewski  or @Zales0123 here 😆

File: docs/bundles/SyliusGridBundle/your_first_grid.rst [hunk 2]
```diff
@@ -51,144 +77,147 @@ That's it. SyliusResourceBundle allows to generate a default CRUD interface incl
...
+    # app/config/grids/admin/supplier.yml
     sylius_grid:
         grids:
-            app_tournament:
-                driver: doctrine/orm # Use appropriate driver.
-                resource: app.tournament
+            app_admin_supplier:
+                    ...
                 sorting:
                     name: asc
```
Comment (tuka217) [inline docs/bundles/SyliusGridBundle/your_first_grid.rst]: Doesn't it expected that name is an array?

File: docs/bundles/SyliusGridBundle/configuration.rst [hunk 3]
```diff
@@ -5,34 +5,30 @@ Configuration Reference
 
     sylius_grid:
         grids:
-            app_user: # Your grid name.
-                driver: doctrine/orm # Data source driver.
-                resource: app.user # Resource name.
+            app_user: # Your grid name
+                driver: doctrine/orm # Data source driver
+                resource: app.user # Resource name
```
Comment (pjedrzejewski) [inline docs/bundles/SyliusGridBundle/configuration.rst]: Should be:

```
driver: 
    name: doctrine/orm
    options:
        class: AppBundle\Entity\User
```

See: https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/AdminBundle/Resources/config/grids/channel.yml#L4

**Comment by pjedrzejewski** [comment]
@ylastapis We will document that as well, but actions need to be tweaked a bit, I will take care of that soon. :)

File: docs/bundles/SyliusGridBundle/configuration.rst (hunk 3 above)
Comment (TheMadeleine) [inline docs/bundles/SyliusGridBundle/configuration.rst]: Fixed now! ;)

**Comment by pjedrzejewski** [comment]
Thank you Magda, nice work!

**Pull request merged**